embedded-kafka/src/main/scala/io/github/embeddedkafka/ops/kafkaOps.scala

```diff
@@ -138,6 +143,32 @@ trait KafkaOps {
     Uuid.randomUuid().toString
   }
 
+  /**
+    * Enhance bootstrap metadata with group.version feature to support the new
+    * consumer group protocol. This uses the standard Kafka API to create
+    * bootstrap metadata with the required features, which is the same approach
+    * used by standard Kafka deployments.
+    *
+    * @param bootstrapMetadata
+    *   the original bootstrap metadata
+    * @return
+    *   enhanced bootstrap metadata with group.version feature
+    */
+  private def enhanceBootstrapMetadata(
+      bootstrapMetadata: BootstrapMetadata
+  ): BootstrapMetadata = {
+    // Use the standard Kafka API (fromVersions) to create bootstrap metadata with features
+    // This is the same approach used by real Kafka deployments
+    val featureVersions = new JHashMap[String, java.lang.Short]()
+    featureVersions.put("group.version", 1.toShort)
```

Comment:
Rather than hard coding this. Any way to use something like "fromKraftVersion(production latest)".

The goal would be to not have to know and hardcode each and every feature and the latest version